packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -19,15 +19,18 @@ export class OverflowSet extends BaseComponent<IOverflowSetProps, {}> implements
       items,
       overflowItems,
       onRenderOverflowButton,
-      className
+      className,
+      focusZoneProps,
+      role = 'menubar'
     } = this.props;
 
     return (
       <FocusZone
+        { ...focusZoneProps }
```

Comment:
Should this be applied before or after the focusZoneProps below?

Reply:
I did that intentionally so that the OverflowSet can enforce the props that it wants for its FocusZone essentially locking down a few of the props.